common/src/main/java/com/microsoft/identity/common/adal/internal/util/StringExtensions.java

```diff
@@ -39,19 +40,21 @@
 import java.security.NoSuchAlgorithmException;
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 import java.util.StringTokenizer;
 
 public final class StringExtensions {
-    /**
-     * The Constant ENCODING_UTF8.
-     */
-    public static final String ENCODING_UTF8 = "UTF_8";
```

Comment:
Really unclear why all constants should be in a single file.  Let me know why you think this is the right thing to do.